fractal_tasks_core/tasks/import_ome_zarr.py

```diff
@@ -100,6 +106,46 @@ def _process_single_image(
             logger=logger,
         )
 
+    # Update Omero-channels metadata
+    if update_omero_metadata:
+        # Extract number of channels from zarr array
+        try:
+            channel_axis_index = image_meta.axes_names.index("c")
+        except ValueError:
+            raise NotImplementedError(
+                f"Existing axes ({image_meta.axes_names}) do not "
+                'include "c".'
```

Comment:
Let's link to the issue 150 here: https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/150
OME-Zarrs without channel axis aren't currently supported in Fractal. We are working on improving our axis flexibility, follow along in https://github.com/fractal-analytics-platform/fractal-tasks-core/issues/150